lib/build/less/components/button.less

```
@@ -190,191 +192,205 @@
+  }
 
-    &:focus {
-        --button--bs-color-outer: var(--focus-ring-error);
-    }
+  &:focus {
+    --button--bs-color-outer: var(--focus-ring-error);
+  }
 
-    &.d-btn--outlined {
-        --button--bc: var(--error-color);
-    }
+  &.d-btn--outlined {
+    --button--bc: var(--error-color);
+  }
 
-    &.d-btn--primary {
-        --button--fc: var(--white);
-        --button--bgc: var(--error-color);
+  &.d-btn--primary {
+    --button--fc: var(--white);
+    --button--bgc: var(--error-color);
 
-        &:hover {
-            --button--bgc: var(--error-color-hover);
-        }
+    &:hover {
+      --button--bgc: var(--error-color-hover);
     }
+  }
 }
 
 //  $$  INVERTED BUTTON
 //  ----------------------------------------------------------------------------
 //  --  Base, clear inverted button
 .d-btn--inverted {
-    --button--fc: var(--white);
-    --button--bgc: transparent;
+  --button--fc: var(--white);
+  --button--bgc: transparent;
 
-    &:hover {
-        --button--fc: var(--white);
-        --button--bgc: hsla(var(--white-hsl) ~" / " 15%);
-    }
+  &:hover {
+    --button--fc: var(--white);
+    --button--bgc: hsla(var(--white-hsl) ~" / " 15%);
+  }
 
-    &:focus {
-      --button--bs-color-outer: var(--focus-ring-inverted);
-    }
+  &:focus {
+    --button--bs-color-outer: var(--focus-ring-inverted);
+  }
 
-    //  --  OUTLINED
-    &.d-btn--outlined {
-        --button--bc: var(--white);
-    }
+  //  --  OUTLINED
+  &.d-btn--outlined {
+    --button--bc: var(--white);
+  }
 
-    //  --  PRIMARY
-    &.d-btn--primary {
-        --button--fc: var(--primary-color);
-        --button--bgc: var(--white);
+  //  --  PRIMARY
+  &.d-btn--primary {
+    --button--fc: var(--primary-color);
+    --button--bgc: var(--white);
 
-        &:hover {
-            --button--fc: var(--primary-color-hover);
-            --button--bgc: hsla(var(--white-hsl) ~" / " 90%);
-        }
+    &:hover {
+      --button--fc: var(--primary-color-hover);
+      --button--bgc: hsla(var(--white-hsl) ~" / " 90%);
     }
+  }
 }
 
-
-//  $$  DISABLED STATES
+//  $$   BUTTON LABEL
 //  ----------------------------------------------------------------------------
-.d-btn {
-    &.d-btn--disabled {
-        pointer-events: none;
-    }
+.d-btn__label {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
 
-    &[disabled],
-    &.d-btn--disabled {
-        color: var(--black-400);
-        background-color: var(--black-100);
-        cursor: not-allowed;
-        border-color: transparent;
-        transition: none;
-    }
+//  $$  BUTTON ICON
+//  ----------------------------------------------------------------------------
+.d-btn__icon {
+  display: flex;
+  margin-top: var(--sun4);
+  margin-bottom: var(--sun4);
+  width: 1em;
+  height: 1em;
+}
+.d-btn__icon--left {
+  margin-left: var(--sun2);
+  margin-right: var(--su4);
+}
+.d-btn__icon--right {
+  order: 1;
+  margin-left: var(--su4);
+  margin-right: var(--sun2);
 }
```

```suggestion
.d-btn__icon {
  display: flex;
  margin-top: var(--sun4);
  margin-bottom: var(--sun4);
  width: 1em;
  height: 1em;
}

.d-btn__icon--left {
  margin-left: var(--sun2);
  margin-right: var(--su4);
}

.d-btn__icon--right {
  order: 1;
  margin-left: var(--su4);
  margin-right: var(--sun2);
}
```

Ensure that you have padding between class definitions

---

Same goes with circle button size selectors

---

and the button icon classes